composer.json

```diff
@@ -69,10 +66,11 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "1.10-dev"
+            "dev-master": "1.11-dev"
         },
         "symfony": {
-            "allow-contrib": false
+            "allow-contrib": false,
+            "require": "^5.4"
```

Comment:
I do not have strong opinion about it but I'm concerned about this change. Until now, it is fine, because we have just one version, so it will protect us from Sf6. It is also recommended to have it in end projects. However, once we will start supporting Sf6, what value should be placed here? We still can discuss it later, once it will be supported, but I just wanted to bring this topic up

Reply:
I don't have a good answer to this one. I'm tempted to keep something here to avoid mixed-version installations, but if that would block GitHub pipelines from even testing with SF6, it wouldn't be good.

That said, Sylius Standard is a template for an end project (this repository is even marked as a template already). As such, I would expect it to follow recommendations for end projects. I would expect people to know enough to be able to change this value in their own downstream repositories, should they want to. And for those who don't know how, or have no preference, I think a sane default provided by us would do just fine (and that sane default would probably be the latest Symfony LTS supported by Sylius).